tests/docker/groovy/configs/groovy_image.yaml

```diff
@@ -0,0 +1,13 @@
+schemaVersion: 2.0.0
+
+metadataTest:
+  entrypoint: [
+    '/usr/bin/java',
+    '-cp',
+    '/app/io_bazel_rules_docker/tests/docker/groovy/libgroovy_image_library-impl.jar:/app/io_bazel_rules_docker/../com_google_guava_guava/jar/guava-18.0.jar:/app/io_bazel_rules_docker/../groovy_sdk_artifact/groovy-2.4.4/lib/groovy-2.4.4.jar:/app/io_bazel_rules_docker/tests/docker/groovy/libgroovy_image.binary-lib-impl.jar:/app/io_bazel_rules_docker/tests/docker/groovy/groovy_image.binary.jar:/app/io_bazel_rules_docker/tests/docker/groovy/groovy_image.binary:/app/io_bazel_rules_docker/tests/docker/groovy/BUILD',
```

Comment:
Can this be broken into multilines?

Reply:
I couldn't find a way to. All the ways to break a string into multilines in a YAML file will either insert a space or a new line char into the string.

Reply:
Ack